libs/permission-checker/tests/Check/Ai/CanCreateAutomationsTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker\Tests\Check\Ai;
+
+use Keboola\PermissionChecker\Check\Ai\CanCreateAutomations;
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+use Keboola\PermissionChecker\Feature;
+use Keboola\PermissionChecker\Role;
+use Keboola\PermissionChecker\StorageApiToken;
+use PHPUnit\Framework\TestCase;
+
+class CanCreateAutomationsTest extends TestCase
+{
+    public static function provideValidPermissionsCheckData(): iterable
+    {
+        yield 'simple token' => [
+            'token' => new StorageApiToken(),
+        ];
+
+        yield 'admin token' => [
+            'token' => new StorageApiToken(
+                role: Role::ADMIN->value,
+            ),
+        ];
+
+        yield 'share token' => [
+            'token' => new StorageApiToken(
+                role: Role::SHARE->value,
+            ),
+        ];
+    }
```

Comment:
Tím, že je za tím ještě furt vrstva sapi, tak asi OK. Když vytvoříme něco, co by nemělo moct třeba měnit projekt, tak to jen uvnitř servicy spadne na SAPI.